File: rev13.py
x = {
	"yellow": "banana",
	"yellow": "peach",
	"green": "banana",
	"orange": "orange",
	"orange": "peach"
}

def rever(x):
	y = {}
	for original_key in x:
		original_value = x[original_key]
		if original_value not in y:
			y[original_value] = [original_key]
		else:
			y[original_value].append(original_key)
	return y

print(rever(x))

File: rev13.js
myDict = {
	"green": "banana",
	"yellow": "peach",
	"blue": "blueberry",
	"green": "avocado"
}


const getRev = (obj) => {
	let reved = {};
	for (let old_key in obj) {
		let old_value = obj[old_key];
		if (old_value in reved == false) {
			reved[old_value] = [old_key]
}
		else {
			reved[old_value].push(old_key)
}
}
	return reved
}


console.log(getRev(myDict))
